scripts/openmc-ace-to-hdf5

```diff
@@ -92,7 +92,7 @@ def ace_to_hdf5(destination, xsdir, xsdata, libraries, metastable, libver):
                 # Add data for new temperature
                 try:
                     print(f"Converting {table.name} (ACE) to {data.name} (HDF5)")
-                    data.add_temperature_from_ace(table, metastable)
+                    data.add_temperature_from_ace(table)
```

Comment:
The `metastable` argument should still be there for normal incident neutron files:
```suggestion
                    if table.data_type == TableType.NEUTRON_CONTINUOUS:
                        data.add_temperature_from_ace(table, metastable)
                    else:
                        data.add_temperature_from_ace(table)
```